server/api.js

```diff
@@ -3,8 +3,14 @@ import db from "./db.js";
 const router = Router();
 
 router.get("/videos", async (_, res) => {
-	const result = await db.query("SELECT * FROM videos");
-	res.json(result.rows);
+	try {
+		const result = await db.query("SELECT * FROM videos");
+		res.json(result.rows);
+	} catch (error) {
+		res
+			.status(500)
+			.json({ success: false, error: "Could not connect to the database" });
```

Comment:
It's  good to see you doing some error handling and sending an error message! Question  for you: Are exceptions always because the database can't be connected  to? 

They are likely to be, of course, but that doesn't mean they always are. Imagine, for instance, that you deleted the videos table by accident. This API  would say it could not connect to the database which could lead to you spending hours troubleshooting  the DB  connection. It doesn't matter so much here but it's a good habit. If you have time try to implement it. 

You can use examples in the the link  to identify  the type of error and return a custom error or you can take the message and class out of the error object and return  these.
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#conditional_catch_blocks